spec.html

```diff
@@ -28,7 +28,7 @@ <h3>Basic Types</h3>
                 <table>
                   <tr>
                     <td class="t">byte</td>
-                    <td>1 byte (8-bits)</td>
+                    <td>1 byte (8-bit signed integer, two's complement)</td>
```

Comment:
Looking at the rest of this page, I don't think this is an accurate change. The grammar uses `byte` to signify like arbitrary byte arrays. A good example is the `byte*12` notation used for ObjectIds.

The signed/unsigned thing really comes into play specifically for the type bytes. I suppose what we need to do instead is define a notation for how to represent signed bytes that act as terminals in the grammar. The real problem is just that "\xFF" isn't a clear way to represent the signed byte -1. I don't know if it's a good idea, but you could invent a notation like `signed_byte(n)` and use it everywhere. You'd have to extend the blurb about how terminals are represented in the grammar to say something like "We use the notation signed_byte(n) to represent a terminal consisting of 8-bit, two's complement signed integer whose value is n".

You could start using this new notation for the null terminators and the binary data subtypes and then get rid of the terminals written as hex strings like "\x01".